.github/workflows/action-build.yml

```diff
@@ -44,6 +44,8 @@ jobs:
         timeout-minutes: 5
         run: |
           make test-${{ inputs.runtime }}
+        env:
+          CARGO_TARGET_DIR: /target
```

Comment:
Was this meant to be `./target`?
Same in L.60

Reply:
No, In CI it is failing to find the test binary with it is `./target`.  Locally it works. But I don't know that this is correct either. There are some nuances around folder layout here (we `cd` to a few folders) that is causing hickups.

Reply:
took different approach